server/server.go

```diff
@@ -577,6 +596,12 @@ func (s *Server) Close() {
 		}
 	}
 
+	if s.tracker != nil {
+		if err := s.tracker.Stop(); err != nil {
+			s.logger.Error("failed to shutdown tracker", "err", err)
+		}
+	}
+
```

Comment:
We don't need termination of Tracker because server does no longer have it.

Reply:
[Resolved](https://github.com/0xPolygon/polygon-edge/pull/386/commits/585d357ff9ff834ac33cc2fd98a060539046f5dc)